build.assets/robotest_run_suite.sh

```diff
@@ -44,9 +44,10 @@ function build_upgrade_step {
   local release=${2:?$usage}
   local storage_driver=${3:?$usage}
   local cluster_size=${4:?$usage}
+  local service_opts='"service_uid":994,"service_gid":994' # see issue #1279
```

Comment:
A total nitpick, but can we please use different numbers for uid and gid? Failure scenarios keep flowing in my head where we'd somehow mix uid/gid in the code and it won't surface if they're the same. Guess I'm a bit paranoid :)

Reply:
Changed to uid:997 gid:994 in a02d745321c709d343af0d0e5a10f9f46bc013d3.  I haven't explicitly tested this beforehand, but I'll keep an eye on the PR build for verification.